RediSearchClient.SampleData/Program.cs

```diff
@@ -5,8 +5,9 @@
 using System;
 using System.Linq;
 using NReJSON;
+using RediSearchClient.SampleData.TestingTypes;
 
-using var muxr = ConnectionMultiplexer.Connect("localhost");
+using var muxr = ConnectionMultiplexer.Connect("192.168.0.59:7000,192.168.0.57:7001,192.168.0.59:7002,192.168.0.59:7003,192.168.0.57:7004,192.168.0.59:7005");
```

Comment:
Let's not change this. It's probably safe to assume that the basic use case here is a single node local instance.

Reply:
Yep, will undo that change

Reply:
Fixed